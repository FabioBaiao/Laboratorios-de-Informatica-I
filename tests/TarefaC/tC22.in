aaa
aaa
aAa
1 2 S
AAL