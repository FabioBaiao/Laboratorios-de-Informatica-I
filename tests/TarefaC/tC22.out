1 0
FIM 3
